securedrop/dockerfiles/focal/python3/Dockerfile

```diff
@@ -31,9 +32,10 @@ RUN curl -s https://openpgpkey.torproject.org/.well-known/openpgpkey/torproject.
 RUN TBB_VERSION=$(curl -s https://aus1.torproject.org/torbrowser/update_3/release/Linux_x86_64-gcc3/x/en-US | grep -oP 'appVersion="\K[^"]*' | head -1) && \
     wget https://www.torproject.org/dist/torbrowser/${TBB_VERSION}/tor-browser-linux64-${TBB_VERSION}_en-US.tar.xz && \
     wget https://www.torproject.org/dist/torbrowser/${TBB_VERSION}/tor-browser-linux64-${TBB_VERSION}_en-US.tar.xz.asc && \
-    gpg2 --verify tor-browser-linux64-${TBB_VERSION}_en-US.tar.xz.asc 2>&1 | grep "Primary key fingerprint:" | sed -e 's/Primary key fingerprint: //' -e 's/ //g' | tail -1 | grep -qE "${TOR_RELEASE_KEY_FINGERPRINT}" && \
+    gpg2 --output ./tor.keyring --export ${TOR_RELEASE_KEY_FINGERPRINT} && \
+    gpgv --keyring ./tor.keyring ~/Downloads/tor-browser-linux64-${TBB_VERSION}_en-US.tar.xz.asc ~/Downloads/tor-browser-linux64-${TBB_VERSION}_en-US.tar.xz && \
```

Comment:
I'm assuming the `~/Downloads/` is an accidental copy-paste from you testing it? That's why CI is failing.

Also, once we've verified it matches, we should delete `tor.keyring`.